roles/pool/config-examples/pool-config-local-tp-example.toml

```
@@ -27,4 +27,4 @@ pool_signature = "Stratum V2 SRI Pool"
 # Local TP (this is pointing to localhost so you must run a TP locally for this configuration to work)
 tp_address = "127.0.0.1:8442"
 shares_per_minute = 1.0
```

let's take the opportunity of this PR and increase this value

1 share per minute is too low for most testing scenarios

```suggestion
shares_per_minute = 6.0
```